src/backend/meet/settings.py

```diff
@@ -23,7 +23,7 @@
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-DATA_DIR = os.path.join("/", "data")
+DATA_DIR = os.environ.get("MEET_DATA_DIR", os.path.join("/", "data"))
```

Comment:
I am not very sure how to prefix this variable. Alternatively, I may remove the prefix (`DATA_DIR`) or prefix with `VISIO` (`VISIO_DATA_DIR`)